src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
```

Comment:
The width and height values are hardcoded. If these values are used multiple times across different classes, consider defining them as variables in 'src/styles/utils/variables.scss' for better maintainability.